frontend/plugins/payment_ui/src/modules/corporateGateway/khanbank/settings/KhanbankColumns.tsx

```
@@ -0,0 +1,122 @@
+  addConfig: (variables: Record<string, any>) => Promise<any>;
+  editConfig: (variables: Record<string, any>) => Promise<any>;
+  removeConfig: (_id: string) => Promise<any>;
+};
+
+export const khanbankColumns = (
+  actions: Actions,
+): ColumnDef<IKhanbankConfigsItem>[] => [
+  {
+    id: 'more',
+    cell: ({ cell }) => {
+      const { t } = useTranslation('payment');
```

ESLint

[error] React Hook "useTranslation" is called in function "cell" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)